[2022-02-12 20:56:41,263] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:50:42.873055+00:00 [queued]>
[2022-02-12 20:56:41,382] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:50:42.873055+00:00 [queued]>
[2022-02-12 20:56:41,383] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:56:41,384] {taskinstance.py:881} INFO - Starting attempt 2 of 2
[2022-02-12 20:56:41,385] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:56:41,410] {taskinstance.py:901} INFO - Executing <Task(PostgresOperator): load_awards> on 2022-02-12T20:50:42.873055+00:00
[2022-02-12 20:56:41,417] {standard_task_runner.py:54} INFO - Started process 5039 to run task
[2022-02-12 20:56:41,470] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', '1_load_initial_data', 'load_awards', '2022-02-12T20:50:42.873055+00:00', '--job_id', '28', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/load_data.py', '--cfg_path', '/tmp/tmpday31eiv']
[2022-02-12 20:56:41,473] {standard_task_runner.py:78} INFO - Job 28: Subtask load_awards
[2022-02-12 20:56:41,543] {logging_mixin.py:112} INFO - Running <TaskInstance: 1_load_initial_data.load_awards 2022-02-12T20:50:42.873055+00:00 [running]> on host f261333f79b3
[2022-02-12 20:56:41,583] {postgres_operator.py:62} INFO - Executing: 
                      COPY dbt_raw_data.awards 
                      FROM '/app/sample_data/raw__awards.csv'
                      DELIMITER ',' CSV HEADER;
                      
[2022-02-12 20:56:41,603] {base_hook.py:89} INFO - Using connection to: id: dbt_postgres_instance_raw_data. Host: postgres-dbt, Port: 5432, Schema: dbtdb, Login: dbtuser, Password: XXXXXXXX, extra: None
[2022-02-12 20:56:41,621] {dbapi_hook.py:176} INFO - 
                      COPY dbt_raw_data.awards 
                      FROM '/app/sample_data/raw__awards.csv'
                      DELIMITER ',' CSV HEADER;
                      
[2022-02-12 20:56:41,624] {taskinstance.py:1150} ERROR - could not open file "/app/sample_data/raw__awards.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/postgres_operator.py", line 65, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 177, in run
    cur.execute(s)
psycopg2.errors.UndefinedFile: could not open file "/app/sample_data/raw__awards.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2022-02-12 20:56:41,627] {taskinstance.py:1194} INFO - Marking task as FAILED. dag_id=1_load_initial_data, task_id=load_awards, execution_date=20220212T205042, start_date=20220212T205641, end_date=20220212T205641
[2022-02-12 20:56:46,192] {local_task_job.py:102} INFO - Task exited with return code 1
